qemu/tests/driver_in_use.py

```diff
@@ -79,13 +79,19 @@ def run_bg_stress_test(bg_stress_test):
 
     def run_subtest(sub_type):
         """
-        Run sub test which include main test and background test.
-        main test: e.g. reboot/shutdown/migration/stop/cont ...
+        Run background test.
```

Comment:
This function need to be renamed. It is similar to  `run_bg_stress_test`, but strictly speaking, it is not ruining '**background**' test as `run_bg_stress_test` did.
Or you probably need to rename both this two functions and their doc string.